docker-compose.test.yaml

```diff
@@ -4,3 +4,4 @@ services:
     app:
         build:
             target: test
+        image: libero/dummy-api_test:${IMAGE_TAG:-master}
```

Comment:
Probably want to push the one built during the smoke tests?

Reply:
I suspect the jobs are all in isolated environments, hence would need to push also the `test` one to avoid re-building it 3 times like in https://travis-ci.com/libero/dummy-api/jobs/157850850

Reply:
Can see what you're doing now. 👍

Though better a different namespace (and the final `prod` image becomes `libero`).

Reply:
...which won't work because forked PRs do not have access to environment variables like `DOCKER_*` credentials

Reply:
Experimenting some more with stages